mayhem/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM fuzzers/go-fuzz:1.2.0 as builder
```

Comment:
Same here, this needs to be something more descriptive than fuzz.go or fuzz.libfuzzer.